sherpa/conftest.py

```diff
@@ -869,3 +869,20 @@ def func(name, mname=None):
         return cls(mname)
 
     return func
+
+
+# Fixtures that control access to the tests, based on the availability
+# of external "features" (normally this is the presence of optional
+# modules). These used to be decorators in sherpa.utils.testing.
+#
+@pytest.fixture
+def requires_pylab():
+    """Runs the test with the pylab plotting backend if available."""
+
+    pylab_backend = pytest.importorskip("sherpa.plot.pylab_backend")
+    plt = pytest.importorskip("matplotlib.pyplot")
+
+    with TemporaryPlottingBackend(pylab_backend.PylabBackend()):
```

Comment:
Shouldn't I have to import this symbol from somewhere?

There's too much magic in pytest!

Reply:
Actually, we already explicitly import this symbol for other pytest fixtures, so it's all good.